app/src/main/java/net/gsantner/opoc/util/FileUtils.java

```diff
@@ -44,20 +47,41 @@ public class FileUtils {
     // Used on methods like copyFile(src, dst)
     private static final int BUFFER_SIZE = 4096;
 
-    public static String readTextFileFast(final File file) {
+    public final static String WITH_BOM = "withBom";
+
+    public static Pair<String, Map<String, Object>> readTextFileFast(final File file) {
+        Map<String, Object> info = new HashMap<>(1);
```

Comment:
This, why does it need to be a hashmap? From my reading, a boolean will do, right?